x/evm/types/params.go

```diff
@@ -153,6 +147,19 @@ func validateEIPs(i interface{}) error {
 		return fmt.Errorf("invalid EIP slice type: %T", i)
 	}
 
+	uniqueEIPs := make([]int64, 0, len(eips))
+	duplicateEIPs := make([]int64, 0, len(eips))
+	for _, eip := range eips {
```

Comment:
Why another for loop? Reuse the existing one

Reply:
addressed 👍 dumb to add a new for loop, it's late.. 😅